cdhweb/pages/templates/cdhpages/page_search.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'base.html' %}
+{% load humanize wagtailcore_tags %}
+
+{% block main %}
+<h1>{{ page_title }}</h1>
+<form action="{% url 'search' %}" method="get">
```

Comment:
will this be a django form once we move this beyond proof of concept?

Reply:
definitely; especially if we later want to get into facets and things like that